src/clr-angular/progress/progress-circle/_progress-circle.clarity.scss

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@function progressNow($base, $value) {
+  // convert to
+  $pr: calc(1 - ($value/100));
+  @return calc($base * $pr);
+}
+@mixin min-height($size) {
+  min-height: $size;
+  min-width: $size;
+  height: $size;
+  width: $size;
+}
+
+$clr-spinner-bg-color: #e6e6e6;
+$clr-spinner-color: #0079b8;
+$clr-spinner-opacity: 1;
+
+$clr-progress-cicle-dasharray: 2 * 3.14 * 56;
```

Comment:
Please either explain why the math needs to be called out here or just use the constant value...